src/main/java/it/pagopa/pn/timelineservice/service/impl/TimelineServiceImpl.java

```diff
@@ -308,6 +312,22 @@ public Mono<NotificationHistoryInt> getTimelineAndStatusHistory(String iun, int
                 .map(notificationStatusInt -> remapTimelineElements(notificationHistoryInt));
     }
 
+    @Override
+    public Mono<DeliveryInformationResponse> getDeliveryInformation(String iun, Integer recIndex) {
+        log.debug("getDeliveryInformation Start - iun={} recIndex={} ", iun, recIndex);
+        return getTimeline(iun, null, false, true)
```

Comment:
Abbiamo necessità che sia strongly?

Reply:
Probabilmente no, può offrirci qualche tutela in casi rari in cui invocano l'api appena ricevuto un messaggio di cortesia e delivery-push-workflow ha impiegato un po' a terminare il flusso di invio presso tutti i canali e quindi ha persistito da poco il PROBABLE_SCHEDULING_ANALOG_DATE.